tests/crashes/140571.rs

```diff
@@ -1,14 +0,0 @@
-//@ known-bug: #140571
```

Comment:
This test was *coincidentally* fixed, but not for a particularly good reason. It's properly fixed in https://github.com/rust-lang/rust/pull/142126.